### PR TITLE
Add endpoint calls to new backend APIs

### PR DESCRIPTION
**Pull request by keyualiceli**

### CODE REVIEW

File: src/utils/API/index.js [hunk 1]
```diff
@@ -112,6 +132,28 @@ export const getApplication = (email, onSuccess, onFailure) => {
...
+      headers: {
+        "Access-Control-Allow-Origin": "*",
+        "Access-Control-Allow-Credentials": true,
+        "x-api-key": apiKey,
+      },
+    };
+  fetch(
+  proxyurl +
+      endpoint +
+      `/applications/application?`+ // questions remain about this endpoint. These new endpoints currently cannot be found in AWS?!
```
Comment (keyualiceli) [inline src/utils/API/index.js:148]: I've been slightly confused about this part. I couldn't find any of the endpoints that Dennis in the backend repo in our AWS. The API Service in AWS had an error for the PUT endpoints 
"The API with ID vewoyuxe51 doesn’t include a resource with path /accept having an integration arn:aws:lambda:us-east-2:885458936997:function:acceptApplications on the PUT method."

Comment (keyualiceli) [reply src/utils/API/index.js]: SOLVED: endpoints in AWS created and connected to the lambda functions

**Review by dunja-tomic — APPROVED** [review]
Looks awesome!

**Comment by wzhang2705** [comment]
Hi Alice

**Review by wzhang2705 — APPROVED** [review]

**Pull request merged**